### DISCUSSION TITLE
FilePicker limit selection count?

### DISCUSSION THREAD

**Discussion by CodingOctocat** [dotnet/maui#24351] [Q&A]
How to use FilePicker and limit selection count?

**Comment by JoaoPedroAlphaSoftware** [comment]
I'm the same. I wish there was a way to control the limit of file selection, but I think the way is to migrate to MediaPicker and perform one image selection at a time.

**Reply by CodingOctocat** [reply]
If there are more files to select and the user needs to select them over and over again, it's a bad experience, maybe it's better to allow multiple selections and then provide a restricted confirmation list page.
